src/components/Welcome.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <div class="container">
+    <h1 class="title_i_promise">I promise</h1>
+    <div class="sheet-wrapper">
+      <v-sheet color="white" elevation="5" height="350" rounded>
+        <div class="button-wrapper">
+          <v-btn
+            @click="redirectToLogInPage"
+            color="deep-purple lighten-2"
+            dark
+            large
+          >
+            {{ titleButtonLogIn }}
+          </v-btn>
+          <v-btn
+            @click="redirectSignUpPage"
+            color="deep-purple lighten-2"
+            dark
+            large
+            outlined
+          >
+            {{ titleButtonSignUp }}
+          </v-btn>
+        </div>
+      </v-sheet>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import router from '@/router';
+
+export default Vue.extend({
+  name: 'Welcome-page',
+
+  data: function () {
+    return {
+      titleButtonLogIn: 'Log in',
+      titleButtonSignUp: 'Sign up',
+    };
+  },
+  methods: {
+    redirectToLogInPage() {
+      router.push('/login');
+    },
+    redirectSignUpPage() {
+      router.push('/signup');
+    },
+  },
+});
+</script>
+
+<style lang="scss">
+.container {
```

Comment:
this is a very generic class name. If you need a container you should use Vuetify's classes instead of writing your own. If you need a very specific class, you should create more specific name

Reply:
Done